web/styling/box-model/borders.md

```diff
@@ -36,15 +38,19 @@ div{
 }
 ```
 
-Or using the shorthand notation, that respects the same order:
+Or using the shorthand notation:
 
 ```css
 div{
-  border: 4px solid red;
+  border: red 4px solid;
 }
 ```
 
-Borders can have different styles, of which the most common is `solid`. Others are `dashed`,`dotted` or `double`.
+You can declare it with any combination of the `width`, `style`, and `color`, as well as in any order.
+
+> ❗ The `style` needs to be set for the border to show. It is because the default value is `none`. 
```

Comment:
let's switch to the active voice

```suggestion
> ❗ You need to set the `style` for the border to show. If you don't, the default value is `none`. 
```

Reply:
I don't know why, I always go for passive 😄